app/controllers/invoices_controller.rb

```diff
@@ -27,17 +27,20 @@ def update
 
   def check
```

Comment:
shouldn't we update this test to reflect these new changes?

Reply:
I didn't get this one, is it missing some spec to be updated?

Reply:
do you see a way to improve these specs? Maybe use another HTTP status code instead of 422, when `all_paid` is false?

https://github.com/comarev/comarev/pull/78/files#diff-ad47999861f867726656425aa35cb97189adbe708c96d572dd7fed2e4cb115d2R126

Reply:
nvm all good, I thought you were asserting the variables declared in the controller